src/chef-server-ctl/plugins/du.rb

```diff
@@ -0,0 +1,25 @@
+require 'mixlib/shellout'
```

Comment:
This file (du.rb) should live in lib/chef_server_ctl/du.rb, under `module ChefServerCtl, because it is not a plugin.